content/en/docs/reference/scheduling/config.md

```diff
@@ -255,7 +255,183 @@ the same configuration parameters (if applicable). This is because the scheduler
 only has one pending pods queue.
 {{< /note >}}
 
-## {{% heading "whatsnext" %}}
+### MultiPoint config
```

Comment:
I'd write:
```suggestion
## Plugins that apply to all extension points {#multipoint}
```
The fragment identifier will then be `multipoint`.

Also, under “Extension points” (line 47), I'd mention and hyperlink to this section.

Reply:
made this `multiple extension points` rather than `all`, think that's clearer